doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -2,111 +2,75 @@
 
 .. _pipeline-tutorial:
 
-#######################
-Running the AP pipeline
-#######################
+.. _pipeline-tutorial-gen3:
+
+###############################
+Running the AP pipeline (Gen 3)
+###############################
 
 Setup
 =====
 
 Pick up where you left off in :doc:`Getting Started <getting-started>`.
-This means you already have a repository of ingested DECam data and have setup
-the LSST Science Pipelines stack as well as ``ap_pipe`` and ``ap_association``.
+This means you already have a repository of ingested DECam data and have set up the LSST Science Pipelines stack.
 
-Your directory structure should look something like
+Your repository should have the following collections, which can be checked using ``butler query-collections <repo>``:
 
-.. code-block:: none
+- **calib/DECam**: biases, flats, defects, camera specs, etc.
+- **raw/DECam**: images to be processesd
+- **refcats**: reference catalogs for calibration
+- **skymaps**: index for the templates
+- **templates/deep**: difference imaging templates
 
-   repo/   <-- where you will be running ap_pipe
-      registry.sqlite3   <--- created during image ingestion
-      image1.fits   <--- linked here by default during image ingestion
-      image2.fits
-      ...
-      ref_cats/   <--- copied or linked here manually by you
-         gaia/
-            shard1
-            shard2
-            ...
-         pan-starrs/
-            shard1
-            shard2
-            ...
-      calibs/
-         calibRegistry.sqlite3   <--- created/updated during calib/defect ingestion
-         cpBIAS/
-            bias1.fits   <--- linked here by default during calib ingestion
-            bias2.fits
-            ...
-         cpFLAT/
-            flat1.fits   <--- linked here by default during calib ingestion
-            flat2.fits
-            ...
-         defects/   <--- put here manually by you prior to defect ingestion
-            defect1.fits
-            defect2.fits
-            ...
-      templates/   <--- copied or linked here manually by you
-         repositoryCfg.yaml
-         deepCoadd/
-            g/
-               0/
-                  psfMatched-0,0.fits
-                  psfMatched-0,1.fits
-                  ...
 
 .. _section-ap-pipe-command-line:
 
 AP pipeline on the command line
 ===============================
 
-The executable to run for the AP Pipeline (`ApPipeTask`) is in ``ap_pipe/bin/ap_pipe.py``.
+Like most Vera Rubin Observatory pipelines, the AP Pipeline is run with an external runner called ``pipetask``.
+This can be found in the ``ctrl_mpexec`` package, which is included as part of ``lsst_distrib``.
+
+The pipeline itself is configured in ``ap_pipe/pipelines/ApPipe.yaml``.
```

Comment:
Please show or otherwise reference an example of what this file should look like, and explain what each part of it does.

Reply:
Is that really in scope for the tutorial, though? I would think most users would use the pipeline as-is (we didn't support code changes to `ApPipeTask`, after all!)

Reply:
It's... borderline 🙂 Could you link to it maybe? It's likely to be some users' first time seeing such a file and it feels like a black box without at least a minimal example.